app/tests/test_admin_page.py

```diff
@@ -342,6 +342,145 @@ def test_add_update_collection_with_provider(self):
             msg="Admin page wrong provider.roles after update"
         )
 
+    def test_add_collection_with_provider_no_description(self):
```

Comment:
Actually this use case is not needed, we cannot have `no description` from the admin interface, but simply a collection empty string.